tools/rpm2img

```diff
@@ -3,22 +3,25 @@
 set -eu -o pipefail
 shopt -qs failglob
 
+OUTPUT_FMT="raw"
+
 for opt in "$@"; do
    optarg="$(expr "${opt}" : '[^=]*=\(.*\)')"
    case "${opt}" in
       --package-dir=*) PACKAGE_DIR="${optarg}" ;;
       --output-dir=*) OUTPUT_DIR="${optarg}" ;;
+      --output-fmt=*) OUTPUT_FMT="${optarg}" ;;
    esac
 done
 
 mkdir -p "${OUTPUT_DIR}"
 
 FILENAME_PREFIX="bottlerocket-${VARIANT}-${ARCH}-${VERSION_ID}-${BUILD_ID}"
-DISK_IMAGE_NAME="${FILENAME_PREFIX}.img.lz4"
-BOOT_IMAGE_NAME="${FILENAME_PREFIX}-boot.ext4.lz4"
-VERITY_IMAGE_NAME="${FILENAME_PREFIX}-root.verity.lz4"
-ROOT_IMAGE_NAME="${FILENAME_PREFIX}-root.ext4.lz4"
-DATA_IMAGE_NAME="${FILENAME_PREFIX}-data.img.lz4"
+DISK_IMAGE_BASENAME="${FILENAME_PREFIX}"
+BOOT_IMAGE_BASENAME="${FILENAME_PREFIX}-boot"
+VERITY_IMAGE_BASENAME="${FILENAME_PREFIX}-root"
+ROOT_IMAGE_BASENAME="${FILENAME_PREFIX}-root"
+DATA_IMAGE_BASENAME="${FILENAME_PREFIX}-data"
```

Comment:
It's weird to me that `VERITY_IMAGE_BASENAME` and `ROOT_IMAGE_BASENAME` have the same value. Maybe incorporate part of the extension?

```suggestion
DISK_IMAGE_BASENAME="${FILENAME_PREFIX}"
DATA_IMAGE_BASENAME="${FILENAME_PREFIX}-data"

BOOT_IMAGE_BASENAME="${FILENAME_PREFIX}-boot.ext4"
ROOT_IMAGE_BASENAME="${FILENAME_PREFIX}-root.ext4"
VERITY_IMAGE_BASENAME="${FILENAME_PREFIX}-root.verity"
```

Then the final extension (`.img.lz4`, `.vmdk`, `.lz4`) would be provided on the same line where we invoke the tool that converts it to that format, which would keep the clarity improvement you're after.

Reply:
I can see how that would be strange. I've gone ahead and amended the commit so that we only use BASENAME for the disk and data images.